daisy_workflows/image_build/windows/post_install.ps1

```diff
@@ -13,18 +13,23 @@ $windows_update_path = 'HKLM:\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate'
 $windows_update_au_path = "$windows_update_path\AU"
 
 function Run-Command {
- [CmdletBinding(SupportsShouldProcess=$true)]
-  param (
-    [Parameter(Mandatory=$true, ValueFromPipelineByPropertyName=$true)]
-      [string]$Executable,
-    [Parameter(ValueFromRemainingArguments=$true,
-               ValueFromPipelineByPropertyName=$true)]
-      $Arguments = $null
-  )
-  Write-Host "Running $Executable with arguments $Arguments."
-  $out = &$executable $arguments 2>&1 | Out-String
-  $out.Trim()
-}
+   param (
+     [Parameter(Mandatory=$true, ValueFromPipelineByPropertyName=$true)]
+       [string]$Executable,
+     [Parameter(ValueFromRemainingArguments=$true,
+                ValueFromPipelineByPropertyName=$true)]
+       $Arguments = $null,
+       [switch]$NoOutString
+   )
+   Write-Host "Running $Executable with arguments $Arguments."
+   if ($NoOutString) {
+    $out = &$executable $arguments 2>&1
```

Comment:
$out = (&$executable $arguments 2>&1).toString() 

As it is, it doesn't handle errors well.
```
$out = Run-Command -NoOutString 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' 'installed'
Running C:\ProgramData\GooGet\googet.exe with arguments -root C:\ProgramData\GooGet installed.
Method invocation failed because [System.Management.Automation.ErrorRecord] does not contain a method named 'Trim'.
At line:17 char:4
+    $out.Trim()
+    ~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : MethodNotFound 

```

Casting it to a string in the assignment, appears to address it well enough.